lips/lip-6.mdx

```diff
@@ -109,14 +108,11 @@ More on these transaction scripts (their specifications, information about error
 
 This section details the workflows or common scenarios related to Validator Set management.
 ## Validator Owner and Validator Operator accounts creation
-LibraRoot (through the Association Operation Services, "AOS") creates Validator Owner and Validator Operator accounts.
-To get an account created, each Validator Owner or Operator needs to provide an authentication key and a name of their organization to LibraRoot. The authentication key is a 32 elements byte array, derived from the public verification key of Ed25519 or MultiEd25519 signature schemes.
+LibraRoot (through the Association Operation Services, "AOS") creates Validator Owner and Validator Operator accounts. To get an account created, each Validator Owner or Operator needs to provide an authentication key and a name of their organization to LibraRoot. The authentication key is a 32 elements byte array, derived from the public verification key of Ed25519 or MultiEd25519 signature schemes.
 
-LibraRoot creates Validator Owner's account by sending the `create_validator_account` transaction script.
-An account address of a Validator Owner is derived from their name. An Owner may ask the LibraRoot to manage the newly created account (in which case for account creation an owner only needs to provide a name), the Owner that established custody of their own keys may later revoke this permission upon notifying the LibraRoot. In the case when LibraRoot maintains ownership of an Owner's account the auth key of that account is set to all-zeroes string and the transactions on behalf of such Owner are sent via AdminScripts.
+LibraRoot creates Validator Owner's account by sending the `create_validator_account` transaction script. An account address of a Validator Owner is derived from their name. An Owner may ask the LibraRoot to manage the newly created account in which case for account creation an owner only needs to provide a name. In the case when LibraRoot maintains ownership of an Owner's account the auth key of that account is set to all-zeroes string and the transactions on behalf of such Owner are sent via AdminScripts.
 
-LibraRoot creates Validator Operator's account by sending the `create_validator_account` transaction script.
-An account addresses of a Validator Operator is derived from their authentication key in the usual way.
+LibraRoot creates Validator Operator's account by sending the `create_validator_account` transaction script. An account addresses of a Validator Operator is derived from their authentication key in the usual way.
```

Comment:
nit: "An account addresses of a Validator Operator" -> "An account address of a Validator Operator"